src/ray/raylet/local_object_manager.cc

```diff
@@ -613,6 +624,8 @@ void LocalObjectManager::RecordMetrics() const {
   ray::stats::STATS_spill_manager_request_total.Record(spilled_objects_total_, "Spilled");
   ray::stats::STATS_spill_manager_request_total.Record(restored_objects_total_,
                                                        "Restored");
+  ray::stats::STATS_spill_manager_request_total.Record(num_failed_deletion_requests_,
```

Comment:
Hmm Wonder if we need more breakdown;;

type: spill/restore/delete
status: success/failed

Reply:
i could work on this as part of 2.2.